docs/elements.md

```diff
@@ -1994,6 +1995,51 @@ The `pl-card` attributes mirror the options of [Bootstrap 4 cards](https://getbo
 
 ---
 
+### `pl-template` element
+
+Displays boilerplate HTML from templates in a reusable way.
+
+#### Sample element
+
+```html
+<pl-template file-name="outerTemplate.mustache" subdirectory="templates">
+  <pl-variable name="is-open">True</pl-variable>
+  <pl-variable
+    name="problem-statement"
+    directory="question"
+    file-name="serverFilesQuestion/statement.html"
+  ></pl-variable>
+</pl-template>
+```
+
+#### Customizations
+
+| Attribute        | Type                                                                                            | Default             | Description                                                                                                  |
+| ---------------- | ----------------------------------------------------------------------------------------------- | ------------------- | ------------------------------------------------------------------------------------------------------------ |
+| `file-name`      | string                                                                                          | -                   | File name of the outer template to use.                                                                      |
+| `directory`      | `question`, `clientFilesQuestion`, `clientFilesCourse`, `serverFilesCourse`, `courseExtensions` | `serverFilesCourse` | Parent directory to locate `file-name`.                                                                      |
+| `warn-undefined` | boolean                                                                                         | false               | Whether to print a warning when rendering templates with undefined variables. Useful for debugging.          |
+| `log-warnings`   | boolean                                                                                         | true                | Whether to log warnings if a rendered template contains elements which are not guaranteed to work correctly. |
```

Comment:
Note to self: Make these names consistent with other warning logging.

Reply:
Done here: https://github.com/PrairieLearn/PrairieLearn/pull/6747/commits/6698f6249a1b8802227ccb4e8ea79618901306c1